language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/extensions/ballerina/document/visitor/UnusedSymbolsVisitor.java

```diff
@@ -187,12 +187,14 @@ public void visit(ModulePartNode modulePartNode) {
     public void visit(ImportDeclarationNode importDeclarationNode) {
         addUnusedImportNode(importDeclarationNode);
 
-        if (importDeclarationNode.moduleName().size() > 0 && importDeclarationNode.moduleName().get(0) != null) {
+        if (importDeclarationNode.moduleName().size() > 0 && importDeclarationNode.moduleName()
+                .get(importDeclarationNode.moduleName().size() - 1) != null) {
```

Comment:
i don't think you have to check null here as you already check for the `size > 0`

Reply:
Agreed, Updated as requested